loader/loader.go

```diff
@@ -83,6 +85,7 @@ func toGunkPackage(fset *token.FileSet, lpkg *packages.Package) (*GunkPackage, e
 		}
 	}
 
+	log.Command("ls", pkgDir, "*.gunk")
```

Comment:
This isn't really a command being run, so I wonder if we should log it at all. `go build -x` does print some commands such as when it creates a directory, but not when it loads Go files or lists directories.

I think the general rule to follow should be "can this action modify something or fail in dozens of ways"? I don't think `ls` falls under that.

Reply:
I thought I saw `go build -x` doing this, but I may have seen it wrong. I'll remove it.

Reply:
I think it's going down a slippery slope - shouldl we also print `cat some-file.gunk` any time we read a Gunk file? :)

Reply:
Yup, of when we write a file. It looks like Go does some of these, but they likely don't all make sense for Gunk.